chain/client/src/stateless_validation/chunk_validator.rs

```diff
@@ -638,3 +650,83 @@ impl Client {
         result.map(|_| None)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use std::collections::HashMap;
+
+    use near_crypto::Signature;
+    use near_primitives::challenge::PartialState;
+    use near_primitives::hash::CryptoHash;
+    use near_primitives::merkle::{Direction, MerklePathItem};
+    use near_primitives::sharding::{
+        ChunkHash, ReceiptProof, ShardChunkHeader, ShardChunkHeaderV3, ShardProof,
+    };
+    use near_primitives::stateless_validation::{
+        ChunkStateTransition, ChunkStateWitness, ChunkStateWitnessInner,
+        MAX_CHUNK_STATE_WITNESS_INNER_SIZE,
+    };
+    use near_primitives::validator_signer::EmptyValidatorSigner;
+
+    use crate::stateless_validation::chunk_validator::validate_chunk_state_witness_size;
+
+    fn dummy_chunk_state_witness() -> ChunkStateWitness {
+        let dummy_chunk_header = ShardChunkHeader::V3(ShardChunkHeaderV3::new(
+            CryptoHash::default(),
+            CryptoHash::default(),
+            CryptoHash::default(),
+            CryptoHash::default(),
+            0,
+            1,
+            0,
+            0,
+            0,
+            0,
+            CryptoHash::default(),
+            CryptoHash::default(),
+            vec![],
+            &EmptyValidatorSigner::default(),
+        ));
+        let dummy_partial_state = PartialState::TrieValues(Vec::new());
+        ChunkStateWitness {
```

Comment:
nit: We have `ChunkStateWitness::empty()`, although there is a comment suggesting it might be removed in the future.

Reply:
Yeah, I had added it for genesis chunk state witness but we can use it for tests